02-peer-protocol.md

```diff
@@ -555,12 +555,12 @@ The funding node:
     - SHOULD send a `closing_signed` message.
 
 The sending node:
-  - MUST set `fee_satoshis` less than or equal to the
- base fee of the final commitment transaction, as calculated in [BOLT #3](03-transactions.md#fee-calculation).
-  - SHOULD set the initial `fee_satoshis` according to its
- estimate of cost of inclusion in a block.
-  - MUST set `signature` to the Bitcoin signature of the close
- transaction, as specified in [BOLT #3](03-transactions.md#closing-transaction).
+  - SHOULD set `fee_satoshis` less than or equal to the base fee of the final
```

Comment:
AFAIU the current spec the MUST sounds more like a mark point to start the closing fee negotiation. It's assuming right now that's the closing fee is more likely to be under commitment tx one (no-HTLC outputs) but in fact last `update_fee`/`commitment_signed` might have been done a while back and feerate already obsolote due to mempool spikes.

So, I would say the benefit of this change is to save in negotiations rounds (and thus reduce the risk of mempool fluctuations during the negotiation falsifying it). 

I believe you should also update the receiving part :
"the base fee of the final commitment transaction as calculated in, MUST fail the connection"

Reply:
> So, I would say the benefit of this change is to save in negotiations rounds (and thus reduce the risk of mempool fluctuations during the negotiation falsifying it).

Not only, right now the spec doesn't let us propose a feerate higher than the commit tx feerate, which is absurd when using anchor outputs.

> we should relax receiver-side first otherwise

That's unnecessary, this condition will only hit with anchor output channels which means the receiver have been upgraded to support anchor outputs, so let's update both at the same time for simplicity.

Reply:
I updated the receiver requirements in https://github.com/lightningnetwork/lightning-rfc/pull/847/commits/bb1012218f72a328e9c945a597ff223a8098d617